scripts/qa_crowbarsetup.sh

```diff
@@ -563,6 +563,12 @@ function get_unclustered_sles12plus_nodes
     echo $sles12plusnodes
 }
 
+function get_non_controller_nodes
+{
+    sles12nodes=get_unclustered_sles12plus_nodes
+    controller_node=get_novacontroller
```

Comment:
@vuntz  Would it be ok to assume that the first value returned by unclustered_sles12plusnodes is always the controller node? and then just remove that value and use the rest as compute? I know it's very hacky but I can't think of something better for now.